docs/src/config.ts

```diff
@@ -79,6 +79,7 @@ export const SIDEBAR: Sidebar = {
       { text: 'Time display', link: 'en/media-time-display' },
       { text: 'Time range', link: 'en/media-time-range' },
       { text: 'Volume range', link: 'en/media-volume-range' },
+      { text: 'Live Indicator/Button', link: 'en/media-live-button' },
```

Comment:
Can this be moved above `Loading Inidcator` to keep this list in alphabetical order?

Reply:
Ah, missed that. Yes. Thanks.